Docker Compose.md

```diff
@@ -0,0 +1,127 @@
+# Docker compose
+
+`docker-compose` is a tool built on top of docker and is used to start multiple service containers and connect them together.
+
+Compose is usually used to create an entire development environment, a typical docker-compose will start up a service for database, backend, frontend, HTTP proxy, etc., connect them all together in a private network and expose public ports on the host.
+
+`docker-compose` can be used for creating production environments, but other solutions are often better suited for that and docker-compose is usually used to simplify setting up the development environment.
+
+Compose configuration is contained within a [docker-compose.yml](#example-docker-compose.yml) script. This script references `Dockerfile`s to be built locally or prebuilt images that will be started as a service. You can specify environment variables to be passed to the container inside of `docker-compose.yml`, volumes, port mappings, etc. For details see the [reference for docker-compose.yml](https://docs.docker.com/compose/compose-file/)
```

Comment:
FROM:
Compose configuration is contained within a [docker-compose.yml](#example-docker-compose.yml) script. This script references `Dockerfile`s to be built locally or prebuilt images that will be started as a service.

TO
Compose configuration is contained within a [docker-compose.yml](#example-docker-compose.yml) script. This script references 'Dockerfiles' which can be built locally or prebuilt images which can be started as a service.